src/frontend/src/app/view/assements/kube-bench-report-list/kube-bench-report-list.component.ts

```diff
@@ -185,19 +195,34 @@ export class KubeBenchReportListComponent implements OnInit {
 
     this.echartsInit('work-node', 'workNodeChart')
     this.echartsInit('k8s-policy', 'k8sPolicyChart')
+    this.echartsInit('control-plane-security', 'controlPlaneSecurityChart')
+    this.echartsInit('control-plane', 'controlPlaneChart')
+    this.echartsInit('etcd-node', 'etcdNodeChart')
+    
+    // this.echartsInit('k8s-policy', 'k8sPolicyChart')
+    // this.echartsInit('k8s-policy', 'k8sPolicyChart')
+
     this.initKubeBenchReportList()
     // this.initKubeBenchReportTypes()
+    // this.getFiveTypeReportUpdateChart()
+  }
+  // Get the latest report update chart for 5 types
+  getFiveTypeReportUpdateChart() {
+    this.echartsLoading = true
     this.getTextTypeTenReports('Worker Node Security Configuration')
-    // this.getTextTypeTenReports('Kubernetes Policies')
+    this.getTextTypeTenReports('Kubernetes Policies')
+    this.getTextTypeTenReports('Control Plane Security Configuration')
+    this.getTextTypeTenReports('Control Plane Configuration')
+    this.getTextTypeTenReports('Etcd Node Configuration')
```

Comment:
As the name is too long and the name on the UI is showing "..."

<img width="940" alt="Screen Shot 2022-12-21 at 21 05 31" src="https://user-images.githubusercontent.com/15973672/208912218-e5d5121b-cd92-4596-957d-3cfc5f6181a9.png">

I would suggest:

Try to show the full name somehow. If you cannot acheive that, change every 'Configuration' to 'Config'.